### DISCUSSION TITLE
Site settings question

### DISCUSSION THREAD

**Discussion by JanOlsmar** [oqtane/oqtane.framework#4664] [Q&A]
The Site Settings in my test site for Oqtane:
![image](https://github.com/user-attachments/assets/bce4e2c1-6ddb-45ea-8def-a604be7dbdc5)

This part does not tell me anything.
Master is not the name  Database. The database is named Oqtane.
Type is not LocalDB
And it is a custom connection.

Have I missunderstood this ...... (too :)

**Comment by sbwalker** [comment]
@JanOlsmar this section actually refers to the Tenant which this Site belongs to. At some point there was a complaint that the concept of Tenant is confusing, so it was changed to Database (as a Tenant in Oqtane is mapped to a specific Database). So Database is actually the Tenant Name, Type is the Database Provider Type (ie. LocalDB, Sql Server, MySQL, SQLite, PostgreSQL), and Connection is the name of the connection string in the appsettings.json file = not the actual connection string details.

**Reply by JanOlsmar** [reply]
Yes I understood. I get LocalDB but it is a SQL server. And text and helptext missleading.
The actual database name had been useful.
